nginx/start.sh

```diff
@@ -2,8 +2,16 @@
 set -e
 
 export IMMICH_WEB_URL=${IMMICH_WEB_URL:-http://immich-web:3000}
+IMMICH_WEB_SCHEME=$(echo "$IMMICH_WEB_URL" | grep -Eo '^https?://' || echo "http://")
```

Comment:
Can you help explaining the changes in this file?

Reply:
This splits the URL(`http://immich-web:3000`) into the scheme(`http://`) and the host part(`immich-web:3000`). The upstream server only allows host/ip + port. I keep the scheme in case the user wants to pass it to a remote machine over HTTPS.

see http://nginx.org/en/docs/http/ngx_http_upstream_module.html#server

Reply:
Basically, to set keepalive you need a nginx server block, and to use a nginx server block you need to be able to use protocol and host separately, so this uses grep to parse out that portion of the url from the original env.

Reply:
Understood, thank you!